browser/css/notebookbar.css

```diff
@@ -351,7 +351,7 @@
 	margin-inline-end: 5px !important;
 	text-align: center;
 }
-.unotoolbutton.notebookbar.has-label > *{ /*so they stay side by side*/
+#zoomreset, .unotoolbutton.notebookbar.has-label > *{ /*so they stay side by side*/
```

Comment:
clever way of doing it : ) without additional JS. Note for myself: Then maybe in the feature we can somehow improve the bigtoolitem or add an additional parameter to apply that while not applying anything else . I need to think about it :thinking: